src/lekcja2_zadania/informacje_o_produktach_zmienne/Product.java

```diff
@@ -0,0 +1,7 @@
+package lekcja2_zadania.informacje_o_produktach_zmienne;
+
+public class Product {
+    String productName;
```

Comment:
nie ma sensu dodawac przedrostka `product` w nazwie tej zmiennej. To ze to jest nazwa produktu wiadomo z kontekstu. wystarczy nazwac to pole `name`

Reply:
ok